### PR TITLE
add bug report issue template

### LINKED ISSUE #3738: Create a GitHub issue template for bugs
We should create an issue template for users asking for help.

We should ask people for information such as

* What kfctl commands did they run
* What is the output of `kfctl version`
* What their kubernetes platform is

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

fix #3738

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3751)

### CODE REVIEW

**Comment by jlewi** [comment]
Thank you so much!
/ok-to-test

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+/kind bug
```
Comment (jlewi) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should we have a list of options like bug, feature request, question?

Comment (yeya24) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Make sense. I have added feature request and question template. PTAL.

**Pull request merged**